test/integration_test/common_test.go

```diff
@@ -928,6 +981,83 @@ func getTokenFromSecret(secretName, secretNamespace string) (string, error) {
 	return "", fmt.Errorf("secret does not contain key 'auth-token'")
 }
 
+func changePxServiceToLoadBalancer(internalLB bool) error {
+	// Check if service is already of type loadbalancer
+	pxService, err := core.Instance().GetService(pxServiceName, defaultAdminNamespace)
```

Comment:
What if Portworx is installed through operator and in `portworx` namespace esp if it is Openshift on AWS? 
Lets make it configurable in the tests to choose which namespace PX can run. And by default we should test it when PX runs in `portworx` namespace, since this is now the default namespace where PX gets installed.

Reply:
Fixed this by adding a new param for PX namespace and defaulting it to `portworx`